src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/idam/IdamTestService.java

```diff
@@ -96,6 +96,16 @@ public User createDefendant(final String letterHolderId) {
         return userService.authenticateUser(email, password);
     }
 
+    private void createUser(CreateUserRequest createUserRequest) {
+        idamTestApi.createUser(createUserRequest);
+        //recommended delay from SIDAM team to stop intermittent auth failures
+        try {
+            Thread.sleep(1000);
```

Comment:
Do we still need this?

Reply:
I dont think so - Ive not seen since tbh. Will close.